### DISCUSSION TITLE
Network: host option in docksal

### DISCUSSION THREAD

**Discussion by prometjohn** [docksal/docksal#1573] [Q&A]
Since this friday, we have had travis builds failing because we are running fin commands like composer install at travis.  Other people have mentioned they have had similar issues with composer install commands on travis within docker.  They mentioned a temporary fix would be appending the --network host option on a docker run command or adding network: host to a service definition in a docker-compose.yml.  But I don't know the equivalent in docksal.yml or wherever I would do that.  Right now we just run fin command for our commands.  Any help would be really appreciated.

**Comment by lmakarov** [comment]
> we have had travis builds failing because we are running fin commands like composer install at travis.

Can you be more specific and provide example from the build error logs? Also, referencing the similar issues and solutions can be helpful.
